vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/payment/payment_util.cc

```diff
@@ -5,7 +5,9 @@
 
 #include "bat/ledger/internal/endpoint/payment/payment_util.h"
 
+#include "base/command_line.h"
 #include "bat/ledger/ledger.h"
+#include "brave/components/brave_rewards/browser/switches.h"
```

Comment:
this is a circular dependency, you cannot include `brave/components/brave_rewards in `bat-native-ledger`

Reply:
resolved: https://github.com/brave/brave-core/pull/7557/files#diff-9709798eaebc36bee441ed094f6ed204130fc6dbfad28a2cc87f10c26772265f